.github/workflows/validate.yaml

```diff
@@ -27,4 +34,8 @@ jobs:
 
     - name: Terraform Format
       id: fmt
-      run: terraform fmt -check -recursive -diff
\ No newline at end of file
+      run: terraform fmt -check -recursive -diff
+
+    - name: Terraform Plan
```

Comment:
This will currently run on pushes to the branch and the pr sync. I think we said we'd run it on the PR sync so that the reviewer can get an idea of the plan change before approving

Reply:
I've added an if statement to check for a PR before running plan